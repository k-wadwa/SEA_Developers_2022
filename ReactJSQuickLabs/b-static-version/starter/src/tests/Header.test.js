import React from 'react';
import Header from '../Components/Header';
import { create } from 'react-test-renderer';

test('Header renders snapshot', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
});
