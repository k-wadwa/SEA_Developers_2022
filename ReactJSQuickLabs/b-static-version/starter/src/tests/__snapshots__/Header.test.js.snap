// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders snapshot 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://qa.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        style={
          Object {
            "width": "100px",
          }
        }
      />
    </a>
    <a
      className="navbar-brand"
      href="/"
    >
      <h1>
        Todo App
      </h1>
    </a>
  </nav>
</header>
`;
